tests/without_mdx_unimoji_extension.py

```diff
@@ -0,0 +1,16 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from unittest import main, TestCase
+
+
+class Test(TestCase):
+    def test_markdown_without_emoji_extension(self):
```

Comment:
Nope. I tried that, but then I started to get errors like this:

```python
ERROR: test_mdx_span_class_option (__main__.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests.py", line 55, in test_mdx_span_class_option
    result = markdown(text, extensions=[UnimojiExtension(span_class='other')])
  File ".../mdx_unimoji.py", line 117, in __init__
    super(UnimojiExtension, self).__init__(*args, **kwargs)
  File ".../lib/python3.5/site-packages/markdown/extensions/__init__.py", line 51, in __init__
    self.setConfigs(kwargs)
  File ".../lib/python3.5/site-packages/markdown/extensions/__init__.py", line 82, in setConfigs
    self.setConfig(key, value)
  File ".../lib/python3.5/site-packages/markdown/extensions/__init__.py", line 73, in setConfig
    value = parseBoolValue(value, preserve_none=True)
  File ".../lib/python3.5/site-packages/markdown/util.py", line 111, in parseBoolValue
    raise ValueError('Cannot parse bool value: %r' % value)
ValueError: Cannot parse bool value: 'other'
```

When I keep the tests in different files and run them separately they pass.